.gitlab-ci.yml

```diff
@@ -98,28 +104,18 @@ build-wasm:
 test:
   stage:                           workspace
   <<:                              *docker-env
-  dependencies:
-    - check-std
   script:
-    - for crate in ${ALL_CRATES}; do
-        cargo test --verbose --all-features --release;
-      done
+    - cargo test --verbose --all-features --release
```

Comment:
Missing the `--all` flag if you want to go without the loop ... however, I really do not recommend doing so. Why have you changed it back?